x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/settings/settings.tsx

```diff
@@ -171,69 +173,64 @@ export const SettingsPage: React.FC<Props> = memo(({ packageInfo }: Props) => {
                   </p>
                 </div>
               ) : (
-                <div>
-                  <EuiTitle>
-                    <h4>
-                      <FormattedMessage
-                        id="xpack.fleet.integrations.settings.packageUninstallTitle"
-                        defaultMessage="Uninstall {title}"
-                        values={{
-                          title,
-                        }}
-                      />
-                    </h4>
-                  </EuiTitle>
-                  <EuiSpacer size="s" />
-                  <p>
-                    <FormattedMessage
-                      id="xpack.fleet.integrations.settings.packageUninstallDescription"
-                      defaultMessage="Remove Kibana and Elasticsearch assets that were installed by this integration."
-                    />
-                  </p>
-                </div>
+                removable && (
+                  <>
+                    <div>
```

Comment:
Would it make sense to use a `<EuiText>` here? That will wrap the `<p>` correctly